app/javascript/components/stylings/buttons/ButtonLink.jsx

```diff
@@ -5,16 +5,30 @@ import PropTypes from 'prop-types';
 
 export default function ButtonLink(props) {
   const navigate = useNavigate();
-  const { to } = props;
+  const {
+    to, className, variant, children,
+  } = props;
```

Comment:
Not big, but Isn't it cleaner if it's one line? ```const { to, className, variant, children } = props;```

Reply:
It's an eslint rule